internal/services/compute/virtual_machine_scale_set_resource.go

```diff
@@ -2294,6 +2306,11 @@ func expandAzureRMVirtualMachineScaleSetExtensions(d *pluginsdk.ResourceData) (*
 			extension.VirtualMachineScaleSetExtensionProperties.AutoUpgradeMinorVersion = &upgrade
 		}
 
+		if u := config["automatic_upgrade_enabled"]; u != nil {
+			upgrade := u.(bool)
+			extension.VirtualMachineScaleSetExtensionProperties.EnableAutomaticUpgrade = &upgrade
+		}
+
```

Comment:
unfortunately this resource is feature-frozen and as such unfortunately we'll need to roll this back:

```suggestion
```